bin/runfx.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+/* eslint-env node */
+
+var Commander = require("commander");
+var Fs = require("fs-promise");
+var Path = require("path");
+var Rimraf = require("rimraf");
+var Util = require("./run-utils");
+var Version = require("../package.json").version;
+var When = require("when");
+
+var DEFAULT_PROFILE = "loop-dev";
+
+Commander
+  .version(Version)
+  .option("-b, --binary <path>", "Path of Firefox binary to use.")
+  .option("-p, --profile <path>", "Path or name of Firefox profile to use.")
+  .option("--binary-args <CMDARGS>", "Pass additional arguments into Firefox.")
+  .parse(process.argv);
```

Comment:
I can't think of a real reason why you'd want to suppress messages from the process landing into your console, so I left it on.